cmd/relayproxy/service/gofeatureflag_test.go

```diff
@@ -44,6 +46,23 @@ func Test_initRetriever(t *testing.T) {
 				Timeout:        20 * time.Millisecond,
 			},
 		},
+		{
```

Comment:
Can you add another test to validate that it works without setting the branch name?